test/functional/interface_zmq.py

```diff
@@ -490,12 +490,6 @@ def test_mempool_sync(self):
         mempool_view = set(mempool_snapshot["txids"])
         get_raw_seq = mempool_snapshot["mempool_sequence"]
         assert_equal(get_raw_seq, 6)
```

Comment:
Thought about this assert being turned into a loop (retrying until `get_raw_seq` is 6), but it doesn't seem to fail currently, and the `sync_all()` call above might be helping to ensure this.

Reply:
Next update

Reply:
is it always `num_txs + 1`? If so, could we assert that instead?

Reply:
`get_raw_seq` is always 6 and `num_txs` is always 5, so we can write it like this. Not sure if this is an improvement but it doesn't hurt so I have changed it.